package.json

```diff
@@ -26,21 +26,21 @@
     "cypress:canary": "cross-env TZ=UTC cypress open --e2e -b chrome:canary",
     "cypress:run": "cypress run",
     "cypress:ci": "yarn cypress:run --config baseUrl=http://localhost:8080 --spec cypress/e2e/smoke/*.cy.js",
-    "serve": "npx -y serve out -p ${REVERSE_PROXY_UI_PORT:=8080}",
+    "serve": "sh -c 'npx -y serve out -p ${REVERSE_PROXY_UI_PORT:=8080}'",
```

Comment:
Why do you need this?

Reply:
Because it was choking on the curly brackets. It seem that they are invalide in their Posix-like interpreter 🤷 
https://yarnpkg.com/features/scripting#script-arguments

that's why I'm starting a shell and passign the command to it.